doc/api.md

```diff
@@ -1847,11 +1850,11 @@ _**Response headers**_
 
 Successful operations return `Content-Type` header with `application/json` value.
 
-#### Modify config group `PUT /iot/services`
+#### Modify config group `PUT /iot/configGroups` or `PUT /iot/services`
```

Comment:
I'd suggest to remove references to `/iot/services` in any file, as it is deprecated feature and, as stated in [doc](https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/deprecated.md)

> Documentation on deprecated features is removed from the repository documentation. Documentation is still available in the documentation set associated to older versions (in the repository release branches).

Reply:
Pending to choose the final term for `configGroups`